gwsumm/data/timeseries.py

```diff
@@ -475,7 +475,8 @@ def get_timeseries_dict(channels, segments, config=GWSummConfigParser(),
             for channel in allchannels:
                 channel = get_channel(channel)
                 ifo = channel.ifo
-                ftype = find_frame_type(channel)
+                ftype = (None if cache and not channel.frametype
+                        else find_frame_type(channel))
```

Comment:
@tjma12, with profuse apologies for how nitpicky this is, can you add an extra leading whitespace here for PEP-8 compliance?

```suggestion
                         else find_frame_type(channel))
```